src/frontend/src/utils/gantt.utils.ts

```diff
@@ -490,13 +490,6 @@ export const GanttWorkPackageStageColorPipe: (stage: WorkPackageStage | undefine
 // maps stage to the desired text color
 export const GanttWorkPackageTextColorPipe: (stage: WorkPackageStage | undefined) => string = (stage) => {
```

Comment:
since the color is white in all cases now, you can just remove this entire colorpipe, and have it just be '#ffffff' wherever this pipe is called